File: package/main/src/Math/valueSwap.ts
/**
 * Swaps two numbers to ensure x < y
 * @param  {number} x First number
 * @param  {number} y Second number
 * @return {[number, number]} Array with numbers in ascending order
 * @example valueSwap(2, 1); // [1, 2]
 * @example valueSwap(1, 2); // [1, 2]
 */
export const valueSwap = (x: number, y: number): [number, number] => {
  return x < y ? [x, y] : [y, x];
};

File: package/umt_python/src/math/value_swap.py
def value_swap(x: float, y: float) -> tuple[float, float]:
    """
    Swaps two numbers to ensure x < y.

    Args:
        x: First number.
        y: Second number.

    Returns:
        Tuple with numbers in ascending order.

    Example:
        >>> value_swap(2, 1)
        (1, 2)
        >>> value_swap(1, 2)
        (1, 2)
    """
    return (x, y) if x < y else (y, x)
